Source/Layout/ASLayout.mm

```diff
@@ -265,7 +261,8 @@ - (ASLayout *)filteredNodeLayoutTree
     }
   }
   
-  ASLayout *layout = [ASLayout layoutWithLayoutElement:_layoutElement size:_size sublayouts:flattenedSublayouts];
+  NSArray *sublayoutsArray = [NSArray arrayWithObjects:flattenedSublayouts.data() count:flattenedSublayouts.size()];
+  ASLayout *layout = [ASLayout layoutWithLayoutElement:_layoutElement size:_size sublayouts:sublayoutsArray];
```

Comment:
IIRC this method is one of our hottest methods, and we copy the resulting array so I'm hopeful this will be a nice bump.